frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -309,10 +299,17 @@ function serializeTrees(trees: Array<Tree>): Array<string> {
 function serializeNodes(nodes: NodeMap): Array<string> {
   return nodes.map((node) => {
     const position = node.position.map(Math.floor);
-    const maybeProperties =
-      (node.additionalCoordinates?.length || 0) === 0
-        ? {}
-        : { additionalCoordinates: JSON.stringify(node.additionalCoordinates) };
+    const maybeProperties = Object.fromEntries(
+      (node.additionalCoordinates || []).map((coord) => [
+        // Export additional coordinates like this:
+        // additionalCoordinate-t="10"
+        // Don't capitalize coord.name, because it it's not reversible for
+        // names that are already capitalized.
+        `additionalCoordinate-${coord.name}`,
```

Comment:
Since `coord.name` is user-controlled it needs to be escaped (could be done in the `serializeTag` method for all attribute names or triggered by a flag argument). I struggled to find the exaxt rules for attribute _names_, but something likes this could be a start:

> The characters that are not allowed in attribute names are space, quotation marks (", '), less than (<), greater than (>), ampersand (&), and some reserved XML characters like question mark (?) and exclamation mark (!).

We would need to test whether saxophone copes with this automatically during deserialization (as it does for attribute values), but probably not :/
It's likely fine to add this to the followup issue if the priority is to get this PR merged.

Reply:
Good point! I added it to #7229 since I think, it is very niche for now.